### ISSUE TITLE
Video sound plays twice and a single instance keeps playing after closing video

### ISSUE THREAD

**Issue by aveexy** [immich-app/immich#7590]
### The bug

If you click a video in the timeline the sound of the video gets played twice, one with a slight delay. After closing the video (via back button) the sound keeps playing, but only a single track. 

It's also possible to have multiple audio tracks from different videos playing after opening multiple videos in sequence.

Seems to only happen on iOS. Android works.

### The OS that Immich Server is running on

Docker on Arch Linux

### Version of Immich Server

1.97.0

### Version of Immich Mobile App

1.97.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
default compose file
```

### Your .env content

```Shell
default env file - nothing special
```

### Reproduction steps

```bash
1. Open a video, same audio plays twice
2. Close video, single audio track keeps playing
3. Open another video, audio track of first video and current video's audio track is playing twice
4. Close video, first and second video's audio track keeps playing
```

**Comment by JimmyGray** [comment]
I'm seeing the same thing. iOS 1.970.0 build.141

**Comment by unshade** [comment]
Same for me on 1.97.0 build 1.41 ; iOS 17.3.1

**Comment by alextran1502** [comment]
Fixed in #7625
